tutorials/2d/using_tilemaps.rst

```diff
@@ -400,7 +403,7 @@ set for the TileSet yet.
 There are 3 kinds of painting modes available for terrain connections:
 
 - **Connect**, where tiles are connected to surrounding tiles on the same
-  TileMap layer.
+  TileMap layer. #CHECK LATER
```

Comment:
Is this supposed to be a comment? In rst that should be:
```suggestion
  TileMap layer. .. CHECK LATER
```

Reply:
Actually I left that for myself as a reminder to review that section, forgot to do that, just updated the PR.